### ISSUE TITLE
Treeview Examples: Hovering to right of tree causes blinking

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#712]
Per feedback from @sh0ji in [issue 225](https://github.com/w3c/aria-practices/issues/225#issuecomment-351110621):

The `hover` class is being added and removed very rapidly when hovering over the `[role=treeitem]`'s section that is outside the width specified in the corresponding CSS selector. The result for the user is white/gray flashing that exceeds 3 flashes per second.

This appears to be because the corresponding span (`firstElementChild` of the tree item) is full-width on mouseover, and then CSS immediately sets it to `16em`, triggering the mouseout event.

```css
[role="treeitem"].hover,
[role="treeitem"] span.hover {
  background-color: #DDDDDD;
  /* this causes the hover class to be removed if the cursor is outside the 16em */
  width: 16em;
}
```

**Comment by mcking65** [comment]
With thanks to @aristov, we have a fix in pull request #689.

**Comment by mcking65** [comment]
Closed with commit d84c8bd.
